ENGAGEHF/Education/Views/VideoPlayers/VideoView.swift

```diff
@@ -0,0 +1,62 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct VideoView: View {
+    let video: Video
+    
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            VideoPlayer(youtubeId: video.youtubeId)
+                .padding(.top)
+            Text(video.description ?? "Video Description")
+                .padding()
```

Comment:
I would suggest that we add a nice rounded rectangle like we do it in the Nourish app around this description. You can take a look how we rendered the description in the video view there which automatically adjusts to the screen size and fills the element accordingly. As with the thumbnail view in the list, I would suggest to package this in an `if let` statement that only show the text if the video description is available and non-empty.